### PR TITLE
Don't hardcoded Datadog API key in the README example

### PR DESCRIPTION
**Pull request by christophetd**

### CODE REVIEW

**Review by alai97 — APPROVED** [review]
Looks good for docs

**Review by jinroh — APPROVED** [review]
Thanks.

File: README.md [hunk 1]
```diff
@@ -29,15 +29,19 @@ module "delegate_role" {
 module "agentless_scanner" {
   source = "git::https://github.com/DataDog/terraform-module-datadog-agentless-scanner"
 
-  api_key               = "YOUR API KEY"
+  api_key               = var.datadog-api-key
   instance_profile_name = module.scanner_role.instance_profile.name
 }
+
+variable "datadog-api-key" {
+
```
Comment (jinroh) [inline README.md:37]: @christophetd would it be possible to do a followup just to mark this example variable as:

```
variable "datadog-api-key" {
  type = string
  sensitive = true
}
```

If it makes sense to you.

Comment (christophetd) [reply README.md:37]: definitely good point

**Pull request merged**